pyamg/dev/elasticity_bar.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
```

Comment:
I think that these two /dev/ examples should go into the pyamg-examples repository